Corona-Warn-App/src/androidTest/java/testhelpers/ScreenShotter.kt

```diff
@@ -36,3 +57,35 @@ inline fun <reified F : Fragment> captureScreenshot(
     launchFragmentInContainer2<F>(fragmentArgs, themeResId, factory)
     takeScreenshot<F>(suffix)
 }
+
+/**
+ * Saves screenshots on the device's sdcard
+ */
+object SDCardCallback : ScreenshotCallback {
```

Comment:
The `screenshots.zip` seems to contain duplicate images:

* In `firebase-screenshots/flame-29-de_DE-portrait/artifacts/sdcard/screenshots`
* In `firebase-screenshots/flame-29-de_DE-portrait/artifacts`

Reply:
Yes, I did not find a nice way to avoid the duplicated images. When I filter them gutil saved the result from both locales into one folder. i.e it does not keep the folder structure from Firebase test results. Which leads to an exception in the command when pulling two files with the same into the same source folder. 
/en_US/Onboarding.png , /de_DE/Onboarding.png. ->/ screenshots/Onboarding.png. if there is away to avoid this easily please let me know?